src/sys/socket/mod.rs

```diff
@@ -601,11 +602,16 @@ pub struct RecvMsg<'a, 's, S> {
 impl<'a, S> RecvMsg<'a, '_, S> {
     /// Iterate over the valid control messages pointed to by this
     /// msghdr.
```

Comment:
Do you think we should document this error case given that it originates from Nix rather than the underlying syscall, users might be confused about it